src/main/java/gregtech/loaders/oreprocessing/ProcessingFineWire.java

```diff
@@ -17,8 +17,8 @@ public ProcessingFineWire() {
 
     public void registerOre(OrePrefixes aPrefix, Materials aMaterial, String aOreDictName, String aModName, ItemStack aStack) {
         if (!aMaterial.contains(gregtech.api.enums.SubTag.NO_SMASHING)) {
-            GT_Values.RA.addWiremillRecipe(GT_OreDictUnificator.get(OrePrefixes.ingot, aMaterial, 1L), GT_Utility.copy(new Object[]{GT_OreDictUnificator.get(OrePrefixes.wireGt01, aMaterial, 2L), GT_Utility.copyAmount(4L, new Object[]{aStack})}), 100, 4);
```

Comment:
How is this a fix?

Reply:
dint feel like looking where the costom fine wire recipes where so i just made it so al of them are the same and always give 8x fine per ingot
why does fine wire even have a difrent amount depending if you can make a wire with it